Serverless/Serverless computing.md

```diff
@@ -0,0 +1,29 @@
+As cloud services became more and more popular, they gave way to a new computing execution model: **serverless**. In this model the cloud provider allocates the necessary resources to run the code we want to run, taking care of everything behind the scenes and allowing the developers to focus more on the development of the business logic.
+
+The term "serverless" might look like a misnomer at a first glance - our code is, after all, executed on some kind of a server - but its point is to describe the developer's perspective. Using this model, the developers and DevOps engineers don't have to be concerned with all the usual server maintenance stuff like allocating CPU and memory, managing the network and its security, and planning the capacity, nor do they have to worry about having more resources than the application requires at a specific point in time. The application uses the resources provided by the cloud service only while it is running, meaning that it can be scaled up or down when needed, or be completely shut down when not in use.
+
+The execution of our code can be triggered by various triggers like an HTTP request, a queue message, changes one the blob storage, or time triggers. All of them will have the same underlying infrastructure and setup, with the only difference being the entry point. 
+
+The main benefits of using serverless solutions are:
+
+- **Writing less code** - The developer can focus on the business logic implementation and utilize the ready-to-use infrastructure features.
+- **Maintaining less infrastructure** - Cloud infrastructure providers manage servers, security, and scaling
```

Comment:
This point doesn't have a dot at the end, let's add it to be consistent with other points.